src_c/polygon.c

```diff
@@ -7,6 +7,43 @@
 #include <stddef.h>
 #include <math.h>
 
+#ifndef M_PI
+#define M_PI 3.14159265358979323846
+#endif
+
+#ifndef M_PI_2
+#define M_PI_2 1.57079632679489661923
```

Comment:
We should rename this to `M_PI_QUO_2` to represent that it's the "quotient" for pi and 2.

Reply:
done